packages/redux/src/bags/reducer.ts

```diff
@@ -1,6 +1,7 @@
 import * as actionTypes from './actionTypes';
```

Comment:
I do not see selectors `areBagPromocodesLoading` and `getBagPromocodesError`. Aren't they necessary as well?

Reply:
This was a mistake. Fixed 👍

Reply:
I still do not see those selectors, have you pushed those changes?